bin/prettier.js

```diff
@@ -50,6 +50,7 @@ if (!filenames.length && !stdin) {
 function getParser() {
   // For backward compatibility. Deprecated in 0.0.10
   if (argv["flow-parser"]) {
+    console.warn("`--flow-parser` is deprecated. Use `--parser flow` instead.");
```

Comment:
As far as I can tell `console.warn` does write to stder..

```
$ cat tmp.js
process.stdout.write('stdout\n')
console.log('console.log')
console.info('console.info')
console.warn('console.warn')
console.error('console.error')
process.stderr.write('stderr\n')
$ node tmp.js | tr '[a-z]' '[A-Z]'
STDOUT
CONSOLE.LOG
CONSOLE.INFO
console.warn
console.error
stderr
```

But I can add a warning for the non-CLI case as well.